flight/Modules/Stabilization/stabilization.c

```diff
@@ -771,7 +786,7 @@ static void stabilizationTask(void* parameters)
 					// Blend from one rate to another. The maximum of all stick positions is used for the
 					// amount so that when one axis goes completely to rate the other one does too. This
 					// prevents doing flips while one axis tries to stay in attitude mode.
-					rateDesiredAxis[i] = rateDesiredAttitude * (1.0f-horizonRateFraction) + rateDesiredRate * horizonRateFraction;
+					rateDesiredAxis[i] = rateDesiredAttitude * (1.0f-horizon_rate_fraction) + rateDesiredRate * horizon_rate_fraction;
```

Comment:
spacing around - operator

Reply:
you always bust me for touching lines with existing problems ;) fixing

Reply:
:D